brainstormforce/ultimate-addons-for-gutenberg
# Fix: Post layout - Conflicting with NexGen.

### Description

Trello - https://trello.com/c/dAWQ0lv9/270-fix-post-layout-nextgen-conflicting-with-post-excerpt

### Screenshots

Issue - https://share.getcloudapp.com/YEuRpnwZ

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Install NextGen Gallery plugin.
- Drag and drop any post layout block and update the page it shows the JSON error.
- Switch to `post-layout-conflicting-with-NexGen` and update the page. It updates successfully.
- Add an excerpt to the post from meta settings and then check the post layout at the front end. (It should display an excerpt in this case)

### Checklist:
